examples/weak_scaling_tester/run.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -e -x
+
+# Get the test location
+TEST_LOCATION="$(pwd)"
+export TEST_LOCATION
+
+# The mapping-tester location
+WEAK_SCALING_TESTER="${TEST_LOCATION}"/../../tools/mapping-scaling-tester/
+
+GENERATOR="${TEST_LOCATION}"/../../tools/mesh-generators/generate_halton_mesh.py
```

Comment:
Is it just using default arguments here? Can you pass the arguments here explicitly?